README.md

```diff
@@ -1,5 +1,20 @@
-# Codidact/Core
-The core Codidact Q&amp;A software implementation.
+<!-- Begin header -->
+
+<p align="center">
+   <img src="https://raw.githubusercontent.com/codidact/landing-page/master/source-assets/img/logo-codidact.png" width="500">
```

Comment:
At least have a `alt=""` here, because apparently screen readers will read the file name otherwise.